<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="iDKELUARGALabel">
       <property name="text">
        <string>ID KELUARGA</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="tAHUNLabel">
       <property name="text">
        <string>TAHUN</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="tAHUNSpinBox">
       <property name="minimum">
        <number>1900</number>
       </property>
       <property name="maximum">
        <number>2100</number>
       </property>
       <property name="value">
        <number>2025</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pENGHASILANBLNLabel">
       <property name="text">
        <string>PENGHASILAN / BLN</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="pENGHASILANBLNLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="pENDIDIKANANAKLabel">
       <property name="text">
        <string>PENDIDIKAN ANAK</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="pENDIDIKANANAKLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="sTATUSRUMAHLabel">
       <property name="text">
        <string>STATUS RUMAH</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="sTATUSRUMAHComboBox">
       <item>
        <property name="text">
         <string>kontrak</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>milik sendiri</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="kATEGORIIPMLabel">
       <property name="text">
        <string>KATEGORI IPM</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="kATEGORIIPMComboBox">
       <item>
        <property name="text">
         <string>rendah</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>menengah bawah</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>menengah atas</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tinggi</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="dIBUATOLEHLabel">
       <property name="text">
        <string>DIBUAT OLEH</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>SIMPAN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>HAPUS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>BERSIH</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>UBAH</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="EditCari"/>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>id_kesejahteraan</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>id_keluarga</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>tahun</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>penghasilan</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>pendidikan_anak</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>status_rumah</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>kategori_ipm</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>dibuat_oleh</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
